src/track/beats.h

```diff
@@ -167,6 +168,9 @@ class Beats : public QObject {
     // have the capability BEATSCAP_SET.
     virtual void setBpm(double dBpm) = 0;
 
+    // Try to round the BPM value to a constant value
```

Comment:
The term "constant" is misleading. Did you mean "integer"? Then the function should be named more explicitly `roundBpmToInteger()`. The term *round* is too generic. Also the functions targets the tempo/bpm property, but not the beat grid as such. 

Please also use *doc comments* `///` for public functions.